lib/benchpark/experiment.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, spec):
         self.helpers = []
         self._spack_name = None
         self._ramble_name = None
+        self.req_vars = ["n_resources", "process_problem_size", "total_problem_size"]
```

Comment:
Please add a device_type variable, which can be set to cpu or gpu.  You can then set it to cpu in  openmp.py, and gpu in rocm.py and cuda.py.  It is possible that we need a mpionly experiment for CPU experiments that don't use openmp......

Reply:
[23c80bc](https://github.com/LLNL/benchpark/pull/750/commits/23c80bc6151a0efebeffba890bfdba7bad0d69cf)